src/style.css

```
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.div {
+  display: block;
+}
+
+.search__form {
+  width: 100%;
+  display: flex;
+  height: 70px;
+  margin: 20px 0;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 3px;
+}
+
+.search__form-sm {
+  height: 42px;
+}
+
+.search__form-input {
+  font-family: inherit;
+  font-weight: inherit;
+  color: #3d4e61;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 100%;
+  width: 100%;
+  border: 0;
+  border-radius: 3px;
+  /* cursor:pointer; */
+}
+
+.search__form-input-sm {
```

we can rename this selector 
```suggestion
.search__input-sm {
```